src/main/resources/public/js/components/store-install-app.js

```diff
@@ -210,6 +211,36 @@ class AppModal extends React.Component {
 
 /** PROPS: app{}, stateApp{}, rateApp(), onInstallButton(), errors[] */
 const AppDescriptionComponent = createClass({
+
+    handleError: function() {
+        let {status, http_status} = this.props.error;
+
+        if(status &&  http_status){
+            let message = '';
+            if(http_status.toString().startsWith('4')) {
+                message = this.context.t('could-not-install-app-400')
+            }else if(http_status.toString().startsWith('5')){
+                message = this.context.t('could-not-install-app-500')
+            }else{
+                message = this.context.t('could-not-install-app')
+            }
+
+            return (
+                <div className="alert alert-danger alert-dismissible" role="alert">
+                    <button type="button" className="close" data-dismiss="alert">
+                        <span aria-hidden="true">&times;</span>
+                        <span className="sr-only">{this.context.t('ui.close')}</span>
+                    </button>
+                    <strong>{this.context.t('sorry')}</strong>
+                    &nbsp;{message}
+                    <br/>
+                    <strong>{this.context.t('error-code')+' : ' + http_status}</strong>
```

Comment:
Do not put the error code into a strong element, it is a technical thing, we want it to be displayed for a better understanding of the problem ... but we don't want users to see only that :)

Maybe you could put it just after the error message, inside parenthesis.